### PR TITLE
WRO-7036: Remove passing `ProgressPlugin` to `VerboseLogPlugin`

### PR DESCRIPTION
**Pull request by hong6316**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
We found that we could access to "ProgressPlugin" component through "compiler.webpack.ProgressPlugin", instead of passing "ProgressPlugin" from CLI as we did in #265.
In this PR, we revert #265 and use dev-utils' ProgressPlugin itself.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- Remove passing "ProgressPlugin" to dev-utils

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-7036

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
     "@babel/preset-react": "^7.16.7",
     "@babel/preset-typescript": "^7.16.7",
     "@babel/runtime": "^7.16.7",
-    "@enact/dev-utils": "^5.0.0-rc.1",
+    "@enact/dev-utils": "enactjs/dev-utils#feature/WRO-7036",
```
Comment (MikyungKim) [inline package.json:52]: Please revert this change.

Comment (hong6316) [reply package.json]: OK

**Review by MikyungKim — APPROVED** [review]
:lgtm:
